app/templates/views/pricing/how-to-pay.html

```diff
@@ -58,7 +58,7 @@ <h2 class="font-body-lg" id="purchase-orders">
     </p>
 
     <p>
-      Your organization may need to <a class="usa-link" href="{{ url_for('main.billing_details') }}">add the Cabinet Office as a supplier</a> before you can raise a <abbr title="purchase order">PO</abbr>.
+      Your organization may need to <a class="usa-link" href="#">add the Cabinet Office as a supplier</a> before you can raise a <abbr title="purchase order">PO</abbr>.
```

Comment:
Flagging this as something we'll have to update further and/or just remove entirely (meaning this whole page).